packages/p2p/src/components/order-details/order-details-wrapper.jsx

```diff
@@ -26,11 +34,7 @@ const OrderDetailsWrapper = ({ children, onPageReturn, page_title }) => {
                     onClick={() => sendbird_store.setShouldShowChatModal(true)}
                 />
             )}
-            renderPageFooterChildren={() =>
-                order_store.order_information.should_show_order_footer && (
-                    <OrderDetailsFooter order_information={order_store.order_information} />
-                )
-            }
+            renderPageFooterChildren={showMobileOrderFooter()}
```

Comment:
Do we need to invoke the function `showMobileOrderFooter` here? Because the MobileFullPageModal invokes it 
```
 {renderPageFooterChildren && (
                    <React.Fragment>
                        {page_footer_parent && (
                            <div
                                className={classNames(
                                    'dc-mobile-full-page-modal__footer-parent',
                                    page_footer_parent_className
                                )}
                            >
                                {page_footer_parent}
                            </div>
                        )}
                        <div className={classNames('dc-mobile-full-page-modal__footer', page_footer_className)}>
                            {renderPageFooterChildren()}
                        </div>
                    </React.Fragment>
                )}
```

Reply:
`MobileFullPageModal` invokes it, but since if I passed in the function reference, `renderPageFooterChildren` would always be truthy by the above conditional statement before the component invokes it, therefore showing the footer even though `should_show_order_footer` is falsy, thus by invoking `showMobileOrderFooter` first it would either return `false` or another function (in this case `footer`, function named since eslint requires all returned components be functionally named), in other words its equivalent to the following statement:
```
<MobileFullPageModal
    ...
   renderPageFooterChildren={order_store.order_information.should_show_order_footer ? () => <OrderDetailsFooter order_information={order_store.order_information} /> : false}
</MobileFullPageModal>
```
In other words its just refactoring the condition into a function and invoking it (any tips/suggestions if this is not favourable would be great 😊)